src/Sylius/Bundle/PayumBundle/Form/Extension/CryptedGatewayConfigTypeExtension.php

```diff
@@ -34,15 +34,15 @@
     /**
      * @param CypherInterface|null $cypher
      */
-    public function __construct(CypherInterface $cypher = null)
+    public function __construct(?CypherInterface $cypher = null)
```

Comment:
Side note: we can make it not accept nulls at all and implement noop cypher as a replacement for null.